test/vcf/test_utils.hpp

```diff
@@ -38,6 +38,7 @@ namespace ebi
       size_t normalized_pos;
       std::string normalized_reference;
       std::string normalized_alternate;
+      vcf::RecordType normalized_alternate_type;
```

Comment:
is this line (and the change in normalize_test.cpp) necessary? the type from TestRecord is not used anywhere I can see.

Reply:
The change in normalize_test.cpp is needed because the constructor for RecordCore class has been changed to use the value directly from the Record class. The change in the struct TestRecord was done so that the change in normalize_test.cpp could be accomodated. 
It should be possible to pass an empty RecordType object in normalize_test.cpp and not change RecordType, but I thought it would lose consistency.

Reply:
mmm, some change was indeed needed in normalize_test.cpp, given that it creates a RecordCore, but TestRecord is not used for anything using the type. TestRecord is meant to lack lots of fields in Record, it just has the fields needed in the normalize_test.cpp. 

Even with your change, `normalized_alternate_type` will always be empty (implicitly), so you can just hardcode an empty string in `test_normalization` as we are hardcoding other fields. Those are clearly mocks and not intended to work outside their narrow tests.